spec/features/blacklight_customizations/accessibility_spec.rb

```diff
@@ -2,42 +2,17 @@
 
 require 'rails_helper'
 
-RSpec.feature "Aria Landmarks", :js do
+RSpec.feature "Aria Landmarks" do
   before do
     visit root_path
     fill_in "q", with: ''
     click_button 'search'
   end
 
-  scenario "should have header landmark" do
+  it "has landmarks" do
```

Comment:
Maybe `aggregate_failures` would be helpful here? 

https://rspec.info/features/3-13/rspec-expectations/aggregating-failures/

Reply:
I think that's just more complexity than necessary. If any one of these assertions fails, it's not drawing the appropriate landmarks.   Maybe I'm not understanding why we'd want to run the rest of the assertions even if one fails.